src/content/decentralized-identity.md

```diff
@@ -0,0 +1,167 @@
+---
+title: Decentralized identity 
+description: What is decentralized identity, and why does it matter?
+lang: en
+template: use-cases
+emoji: ":id:"
+sidebar: true
+sidebarDepth: 2
+summaryPoint1: Traditional identity systems have centralized the issuance, maintenance and control of your identifiers.
+summaryPoint2: Decentralized identity solutions give the power back to the individual, further decreasing reliance on centralized third-parties.
+summaryPoint3: Thanks to crypto, users now have the tools to issue, hold and control their own identifiers and attestations once again. 
```

Comment:
Users can't issue their attestations (well they can, but self-issued attestations in theory should be meaningless)

Reply:
Oh, I think the "issue" part only has to do with decentralized identifiers. A possible suggestion would be to  rework the sentence to read: "Thanks to crypto, users can now issue identifiers, and hold and control attestations." Does this work?